app/src/androidTest/java/com/github/sdp_begreen/begreen/models/PhotoMetadataTest.kt

```diff
@@ -0,0 +1,74 @@
+package com.github.sdp_begreen.begreen.models
+
+import android.graphics.Bitmap
+import android.os.Parcel
+import com.github.sdp_begreen.begreen.matchers.ContainsPropertyMatcher.Companion.hasProp
+import org.hamcrest.CoreMatchers.*
+import org.hamcrest.MatcherAssert.assertThat
+import org.junit.Test
+import java.util.*
+
+class PhotoMetadataTest {
+    @Test
+    fun photoParcelConstructorNotNull() {
+        val photoMetadata = PhotoMetadata(Parcel.obtain())
+        assertThat(photoMetadata, notNullValue())
+    }
+
+    //@Test
```

Comment:
Add a comment explaining why this is commented out?

Reply:
Yes you're right I added a comment in the `MainActivityTest` for similar case on bitmap comparison, but not in this one, it's an oversight on my part. But I will uncomment it and compare bitmap width and height as you suggest it in my next PR.